recipes/r-lasr/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "lasr" %}
+{% set version = "0.13.7" %}
+{% set posix = "m2-" if win else "" %}
+
+package:
+  name: r-{{ name }}
+  version: '{{ version }}'
+
+source:
+  url: https://r-lidar.r-universe.dev/src/contrib/lasR_{{ version }}.tar.gz
+  sha256: 746e84837af8a2d294321b76c790be5f0d57fc9e305a2e0f72703926a68f64b7
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
```

Let's add this boilerplate preemptively in case anyone wants to attempt Windows builds later:

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'        # [win]
    - '*/Rblas.dll'    # [win]
    - '*/Rlapack.dll'  # [win]
```